### ISSUE TITLE
Email notification to application when map pack is ready

### ISSUE THREAD

**Issue by CrispinF** [exeGesIS-SDM/ELM#37]
Triggered on completion of #18 

I think Sarah would like the pack document to be attached to the email, but this could cause problems with attachment sizes, so I'd say this could be tried but needs test/review. If this does work, it wouldn't negate the need for download page, as emails are inherently unreliable.

**Comment by sarahpoppy** [comment]
If the email just contains a link to the document, that's fine as well if it makes life much easier.   The key requirement being that the customer is notified that it is available and can access easily.

**Comment by CrispinF** [comment]
@sarahpoppy a link in the email is possible, and avoids attachment issues.

The quickest solution will be a link to the "retrieve hefer" page with the SBI as a query parameter, so the user will see the document download immediately without having to key in their SBI again. However they will need to be authenticated. Is this acceptable?

Linking to the document itself would need a bit more development (again noting that this document access has to be secured - it cannot just be a plain hyperlink to the PDF, or anyone would be able to access anybody's packs).

Could you assign a priority to this issue?

**Comment by sarahpoppy** [comment]
Essential for the email, low for attachment

The link that current CS customers get is a randomly generated one so that people can't guess other customers packs and people don't need to be authenticated - is that not an option?

**Comment by CrispinF** [comment]
Yes that is an option, but it's not really secure, and it's more work. Since in this case the applicant has to register and sign in, it seems reasonable they need to be signed in to collect their pack, then we can ensure zero risk of inappropriate access.

**Comment by sarahpoppy** [comment]
Fair enough, happy with that.

**Comment by epennifold** [comment]
Agreed in review meeting with SP that email should be sent when SBI is not valid (does not find parcels).
